src/extension/openFileAtLocation.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {RemoteExtension} from "../common/remoteExtension";
+import {ReactNativeProjectHelper} from "../common/reactNativeProjectHelper";
+import {InternalErrorCode} from "../common/error/internalErrorCode";
+import {ErrorHelper} from "../common/error/errorHelper";
+import * as path from "path";
+import * as Q from "q";
+
+/* Usage:
+...path\openFileAtLocation.js filename:lineNumber
+...path\openFileAtLocation.js filename
+*/
+
+if (process.argv.length < 3) {
+    throw "Wrong number of parameters provided. Please refer to the usage of this script for proper use.";
+}
+
+const fullpath = process.argv[2];
+const dirname = path.normalize(path.dirname(fullpath));
+
+// In Windows this should make sure c:\ is always lowercase and in
+// Unix '/'.toLowerCase() = '/'
+const normalizedDirname = dirname.charAt(0).toLowerCase() + dirname.slice(1);
```

Comment:
Yes, yesterday while testing it in windows I found out that vscode manages the workspace with `c` and RN gives it with `C`... Usually this shouldn't matter, because windows support both, but VSCode hashes the workspace path in order to identify it, so by having that one character different the whole hash is different.